apps/dashboard/src/components/primitives/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const functionIcon =
```

Comment:
base64 svg icons, more about that in comments below :P

Reply:
~Do they have to be in base 64? Can we set them in an SVG Icon map?~

I just figured out the strategy. Smart! Please add a comments explaining it for the future reader.

Reply:
`...more about that in comments below :P`